docs/concepts/providers.mdx

```diff
@@ -50,7 +50,7 @@ In addition to on-lattice providers, some capability contracts have implementati
 
 The list of capability providers continues to grow. Please refer to these links for more information:
 
-- [first-party providers](https://github.com/wasmCloud/wasmCloud/tree/main/crates/providers) - a list of supported capability providers
```

Comment:
Shouldn't this be main for 1.0 docs?

Reply:
@ricochet Yeah, I pointed to 0.82 as a stopgap since that link no longer worked, but this question made me realize the providers moved permanently. Adjusting!